package.json

```diff
@@ -43,29 +43,47 @@
     "@types/through2": "^2.0.33",
     "chai": "*",
     "codecov": "^3.1.0",
+    "download": "^7.1.0",
     "eslint": "^6.0.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-node": "^9.0.0",
     "eslint-plugin-prettier": "^3.0.0",
     "execa": "^2.0.0",
+    "file-loader": "^4.0.0",
     "fs-extra": "^8.0.0",
     "google-proto-files": "^1.0.0",
     "gts": "^1.0.0",
     "intelli-espower-loader": "^1.0.1",
+    "is-docker": "^2.0.0",
+    "json-loader": "^0.5.7",
+    "karma": "^4.1.0",
+    "karma-child-process": "^1.0.3",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage": "^1.1.2",
```

Comment:
fun challenge if you got bored, it would be nice to submit a patch to make `karma-coverage` use [nyc](https://www.npmjs.com/package/nyc), as Istanbul is not actively supported any more.